molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -42,6 +43,10 @@
     <#if app_settings.cssHref?has_content>
         <link rel="stylesheet" href="<@resource_href "/css/${app_settings.cssHref?html}"/>" type="text/css">
     </#if>
+    <#if !version?? || version == 1>
```

Comment:
So we are not loading the polyfills for Vue plugins.
The Menu.vue will do it for us?

Reply:
That's up to the version > 1 code that includes the header. Currently polyfills are imported in vue plugins.